src/main/java/emissary/core/IBaseDataObjectXmlHelper.java

```diff
@@ -113,10 +71,6 @@ public final class IBaseDataObjectXmlHelper {
      * The IBaseDataObject set method name for Data.
      */
     public static final String DATA_SET_METHOD_NAME = "setChannelFactory";
```

Comment:
We should consider moving the method name constants to an `emissary.core.constants.IBDOMethodNames` class and moving the element name constants to an `emissary.test.core.constants.AnswerFileElementNames` class.